tests/scripts/thread-cert/thread_cert.py

```diff
@@ -111,6 +111,10 @@ def __init__(self, *args, **kwargs):
         self._do_packet_verification = PACKET_VERIFICATION and hasattr(self, 'verify') \
                                        and self.PACKET_VERIFICATION == PACKET_VERIFICATION
 
+        # store all the backbone network names that are used in the test case,
+        # it keeps empty when there's no backbone traffic in the test (no otbr or host nodes)
```

Comment:
I think the comment isn't necessary. We can remove it.

Reply:
define  `_backbone_network_names` as a member variable in the `TestCase` class might be somehow strange as not all the test case has backbone traffic, I would like to put some note here to people who may have some confuse. Thoughts?